apps/frontend/src/app/5_pages/ZeroPage/ZeroPage.tsx

```diff
@@ -62,63 +65,204 @@ export const ZeroPage: FC = () => {
     }
   }, [account, liquity]);
 
+  const handleTroveSubmit = useCallback(
+    async (value: SubmitValue) => {
+      const wallet = getWallet(0);
+
+      if (wallet?.provider) {
+        const { address, abi } = await getContract(
+          'borrowerOperations',
+          'zero',
+          getRskChainId(),
+        );
+
+        const contract = new Contract(
+          address,
+          abi,
+          new providers.Web3Provider(wallet.provider).getSigner(),
+        );
+
+        if (trove?.debt?.gt(0)) {
+          const params: Partial<TroveAdjustmentParams<Decimalish>> = {};
+
+          if (value.borrow) {
+            params.borrowZUSD = value.borrow;
+          }
+
+          if (value.repay) {
+            params.repayZUSD = value.repay;
+          }
+
+          if (value.depositCollateral) {
+            params.depositCollateral = value.depositCollateral;
+          }
+
+          if (value.withdrawCollateral) {
+            params.withdrawCollateral = value.withdrawCollateral;
+          }
+
+          const adjustedTrove = await adjustTrove(
+            wallet.accounts[0].address,
+            params,
+          );
+          setTransactions([
+            {
+              title: 'Adjusting Trove',
+              contract,
+              fnName: 'adjustTrove',
+              config: {
+                value: adjustedTrove.value,
+                gasLimit: '650000',
```

Comment:
If you want to customize gas limits, we should create a global config file. It should be also discussed with the team.

cc @soulBit